admin_manual/installation/linux_installation.rst

```diff
@@ -2,46 +2,59 @@
 Preferred Linux Installation Method
 ===================================
 
+We recommend the installation from the tar ball for large productive environments 
```

Comment:
And to tidy that up exactly in English, either:
"For production environments, ..."
or
"For a production environment, ..."